src/test/custom_loader_test.cpp

```diff
@@ -246,6 +306,12 @@ class TestCustomLoader : public ::testing::Test {
         std::filesystem::create_directories(cl_model_1_path);
     }
     void TearDown() {
+        std::string configStr = empty_config;
+        // Create config file
+        std::string fileToReload = cl_models_path + "/cl_config.json";
+        createConfigFileWithContent(configStr, fileToReload);
+        ovms::ModelManager& manager = ovms::ModelManager::getInstance();

```

Comment:
Since each unit test should be separated from each other and do not rely on previous test behavior (tests can be executed in any order, can also be executed in parallel) we should mock ModelManager separately from singleton instance. This is why in most cases we use `ConstructorEnabledModelManager` class from test_utils.hpp. Please check our other unit tests.